### PR TITLE
Restore twitter connector

### PR DESCRIPTION
**Pull request by VictorVelarde**
Restore twitter connector for those who manage their own credentials

## Steps for acceptance

- staging: https://twitterdata.carto-staging.com/dashboard/datasets/
- New Dataset / Twitter
- Choose a small period (4h) and a bit weird term: eg 'a-twitter-search'. Set the credits slider to the minimum
image
- Connect dataset

You will see:
- a successful import text
- a working SHOW button
- a table with results

### CODE REVIEW

File: lib/assets/javascripts/cartodb/common/dialogs/create/listing/import_options.js [hunk 1]
```diff
@@ -125,8 +125,8 @@ module.exports = {
   },
   Twitter: {
     ClassName: ImportTwitter,
-    enabled: function (config, userData) {
-      return userData.get('twitter').enabled && !!config.get('datasource_search_twitter');
+    enabled: function (_, userData) {
+      return userData.get('twitter').enabled;
```
Comment (jesusbotella) [inline lib/assets/javascripts/cartodb/common/dialogs/create/listing/import_options.js:129]: This is for editor, but I guess that it's good to have consistent code 😁

File: lib/assets/javascripts/builder/components/modals/add-layer/content/imports/imports-selector/imports-selector-model.js [hunk 2]
```diff
@@ -86,11 +86,10 @@ module.exports = Backbone.Model.extend({
   },
 
   _isTwitterImportEnabled: function () {
-    if (!this._configModel.get('datasource_search_twitter') ||
-        !this._userModel.get('twitter').enabled ||
-        !this._userModel.canCreateTwitterDataset()) {
+    if (!this._userModel.get('twitter').enabled) {
```
Comment (jesusbotella) [inline lib/assets/javascripts/builder/components/modals/add-layer/content/imports/imports-selector/imports-selector-model.js:89]: I guess we could just return `this._userModel.get('twitter').enabled`.

Comment (VictorVelarde) [reply lib/assets/javascripts/builder/components/modals/add-layer/content/imports/imports-selector/imports-selector-model.js:89]: 👌  I removed conditions and I didn't notice it

**Comment by VictorVelarde** [comment]
Thx for the comments, 

Yeah, the date picker appeared broken, like this:
image

I've applied your suggestions

**Pull request merged**